app/frontend/components/taskList/taskRow.tsx

```diff
@@ -0,0 +1,76 @@
+import { ErrorMessage } from '@hookform/error-message'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useForm } from 'react-hook-form'
+import { TaskFragment, TaskInput, useTaskUpdateMutation } from '../../generated/graphql'
+import { InputField } from '../inputField/inputField'
+import { taskInputSchema } from '../taskDetailsModal'
+
+interface TaskRowProps {
+  task: TaskFragment
+}
+
+type TaskRowFormData = Pick<TaskInput, 'title'>
+
+export const TaskRow = ({ task }: TaskRowProps) => {
+  const [{ fetching }, taskUpdate] = useTaskUpdateMutation()
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<TaskRowFormData>({
+    mode: 'onChange',
+    defaultValues: {
+      title: task.title,
+    },
+    resolver: yupResolver(taskInputSchema),
+  })
+
+  const handleSubmitTask = async (taskData: TaskRowFormData) => {
+    try {
+      const result = await taskUpdate({
+        id: task.id,
+        data: {
+          projectId: task.project.id,
+          title: taskData.title,
+        },
+      })
+      if (result.error) {
+        throw new Error(`GraphQL Error ${result.error}`)
+      }
+      reset()
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error)
+    }
+  }
+
+  return (
+    <div className="flex w-[500px]">
+      <InputField
+        variant="primary"
+        className="font-small dark:placeholder-grey ml-2 rounded read-only:bg-gray-100 read-only:opacity-50 dark:border-white dark:bg-slate-800 dark:text-white"
```

Comment:
Do we need those tailwind classes?
I would let the `InputField` component do the styling.

Reply:
because ```InputField``` doesn't support dark theme

Reply:
But then the `InputField` component would be the right place to fix that.
We have the component to keep this somewhat consistent. 😊

Reply:
i checked all places where we use ```InputField``` and everywhere set colors for dark theme

Reply:
I think Pascal wanted the className values inside the InputField component and not setting it in usage of InputField